From 859fad15a43008b32020c8f445a7c2b490832fa1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Fri, 31 Oct 2025 12:30:01 +0100
Subject: [PATCH] Support dark mode

This ensures that the application follows the system color scheme.
---
 shell/cc-application.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/shell/cc-application.c b/shell/cc-application.c
index b7cee6ec3..9a909c437 100644
--- a/shell/cc-application.c
+++ b/shell/cc-application.c
@@ -231,6 +231,8 @@ cc_application_startup (GApplication *application)
   G_APPLICATION_CLASS (cc_application_parent_class)->startup (application);
 
   hdy_init ();
+  hdy_style_manager_set_color_scheme (hdy_style_manager_get_default (),
+                                      HDY_COLOR_SCHEME_PREFER_DARK);
 
   // Initialize keyfile with the provided path
   if (!initialize_keyfile()) {
